### PR TITLE
Bump zwave-js to 8.0.6

### PR DESCRIPTION
**Pull request by onkytonk**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @onkytonk, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;addons#2132).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by raman325 — APPROVED** [review]

**Comment by raman325** [comment]
Changelog: https://github.com/zwave-js/node-zwave-js/releases/tag/v8.0.6

**Comment by MartinHjelmare** [comment]
@onkytonk thanks for the PR. Did you test the updated add-on locally?

File: zwave_js/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 0.1.33
```
Comment (MartinHjelmare) [inline zwave_js/CHANGELOG.md:3]:
```suggestion
## 0.1.33

```

File: zwave_js/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 0.1.33
+
+- Bump Z-Wave JS to 8.0.6
+
 # 0.1.32
```
Comment (MartinHjelmare) [inline zwave_js/CHANGELOG.md:7]:
```suggestion
## 0.1.32

```

**Comment by onkytonk** [comment]
> @onkytonk thanks for the PR. Did you test the updated add-on locally?

I'm not sure how to do that, otherwise I would have. This is my first PR with home assistant, and my second PR ever, so if you have any guidance, that would be appreciated.

**Comment by MartinHjelmare** [comment]
The easiest way in this case where we need access to the Z-Wave USB port is to copy the updated add-on to your computer running Home Assistant over ssh. Then remove the `"image"` key in the config.json file for the add-on. Then build the add-on locally by installing that local version in the Supervisor add-on store. You may need to reload the store to have the local add-on show in the store.

https://developers.home-assistant.io/docs/add-ons/testing#remote-development

**Comment by onkytonk** [comment]
> The easiest way in this case where we need access to the Z-Wave USB port is to copy the updated add-on to your computer running Home Assistant over ssh. Then remove the `"image"` key in the config.json file for the add-on. Then build the add-on locally by installing that local version in the Supervisor add-on store. You may need to reload the store to have the local add-on show in the store.
> 
> https://developers.home-assistant.io/docs/add-ons/testing#remote-development

Awesome - Thanks for your guidance! 

I managed to get it installed and working on my HA. I know its working because I had a device that was missing in the previous release, that I created a PR for and its now recognised and working too! 

Everything else is working as it was previously.

**Review by MartinHjelmare — APPROVED** [review]
Great, thanks!

**Pull request merged**